Makefile

```diff
@@ -163,9 +166,21 @@ PREFIX                    ?=
 BIN_PATH                  ?=$(PREFIX)/usr# /bin is appended later
 BIN_AFTER_INST_PATH       ?=$(BIN_PATH)# needed for debian package and desktop file exec
 PROMPT_BIN_PATH           ?=$(PREFIX)/usr# /bin is appended later
+ifdef MINGW32
+LIB_PATH 	           	  ?=$(PREFIX)/mingw32/lib/
+LIBDEV_PATH 	       	  ?=$(PREFIX)/mingw32/lib/
+INCLUDE_PATH         	  ?=$(PREFIX)/mingw32/include/
+else 
+ifdef MSYS
+LIB_PATH                  ?=$(PREFIX)/usr/lib
+LIBDEV_PATH               ?=$(PREFIX)/usr/lib
+INCLUDE_PATH              ?=$(PREFIX)/usr/include
+else
```

Comment:
Does `oidc-token` use the lib we build here? Why not? Would it be possible? I think it is cleaner to use the lib, similarly to what we do on other systems

Reply:
`oidc-token` does not use the library built for `mingw32`  target. The reason is that `oidc-token` is built for `MSYS` target, so I wanted to keep the consistency -> library is from the same target. But, it's of course possible to link the library from `mingw32` to `oidc-token` for `MSYS` env.